lib/crypto/aead.lua

```diff
@@ -0,0 +1,132 @@
+--
+-- SPDX-FileCopyrightText: (c) 2025 jperon <cataclop@hotmail.com>
+-- SPDX-License-Identifier: MIT OR GPL-2.0-only
+--
+
+--- AEAD (Authenticated Encryption with Associated Data) operations.
+-- This module provides a Lua wrapper for AEAD operations,
+-- using the underlying 'crypto_aead' C module.
+--
+-- @module crypto.aead
+
+local new = require("crypto_aead").new
+local sub = string.sub
+
+--- AEAD instance methods.
+-- This table represents an AEAD instance configured for a specific algorithm.
+-- It is created by calling the main function of the `crypto.aead` module.
+-- @type AeadCipherInstance
```

Comment:
from where this format come from? I'm not against, but we should discuss if we gonna use it for Lunatik as whole.. I wouldn't do this in a single module only..

Reply:
From [LDoc](https://stevedonovan.github.io/ldoc/manual/doc.md.html).